src/components/User/README.md

```diff
@@ -59,9 +61,18 @@ LANDING_BLOCK-->
 | avatar          | User avatar                             | [AvatarProps](../Avatar/README.md#properties) `React.ReactElement` |         |
 | name            | User name                               |                         `React.ReactNode`                          |         |
 | description     | User description                        |                         `React.ReactNode`                          |         |
-| size            | User block size                         |                  `'xs'` `'s'` `'m'` `'l'` `'xl'`                   |   `m`   |
+| size            | User block size                         |              `'2xs'` `'xs'` `'s'` `'m'` `'l'` `'xl'`               |   `m`   |
 | aria-label      | `aria-label` for user block             |                              `string`                              |         |
 | aria-labelledby | `aria-labelledby` for user block        |                              `string`                              |         |
 | className       | Custom CSS class for root element       |                              `string`                              |         |
 | style           | HTML style attribute                    |                       `React.CSSProperties`                        |         |
 | qa              | HTML `data-qa` attribute, used in tests |                              `string`                              |         |
+
+## CSS API
+
+| Name                         | Description                                                |
+| :--------------------------- | :--------------------------------------------------------- |
+| `--g-user-font-size`         | Name and description font size for `size='s'` and larger   |
+| `--g-user-line-height`       | Name and description line height for `size='s'` and larger |
+| `--g-user-small-font-size`   | Name font size for `size='xs'` and smaller                 |
+| `--g-user-small-line-height` | Name line height for `size='xs'` and smaller               |
```

Comment:
CSS API is designed to ignore all props, so only 2 vars are relevant: `--g-user-font-size`, `--g-user-line-height`. There should not be CSS API for props variants